packages/cli/src/bin/ceramic.ts

```diff
@@ -6,7 +6,7 @@ program
     .command('daemon')
     .option('--ipfs-api <url>', 'The ipfs http api to use')
     .option('--ethereum-rpc <url>', 'The Ethereum RPC URL used for communicating with Ethereum blockchain')
-    .option('--anchor-service-api <url>', 'The anchor service URL to use')
+    .option('--anchor-service-url <url>', 'The anchor service URL to use')
```

Comment:
Is there a reason you renamed this? Seems like the param name should make this clear enough.

Reply:
@oed Actually I had been feeling like these flags should be more descriptive.. "ipfs-api-url" and "anchor-service-api-url" leave little room for doubt.

Reply:
But isn't it redundant when the parameter name is `<url>` anyway?

Reply:
Yeah not a huge deal either way but a bit easier to remember. For me I may forget what the flag accepts (e.g. port number, full url, version?) so I would have to take an extra step and do `--help` to be sure.

Reply:
I also prefer including 'url' in the name so it's clearer.  I'd go with either `anchor-service-url` or `anchor-service-api-url` personally.  But I also don't feel strongly either way and am happy to go with whatever you think is best @oed

Reply:
As long as all options are consistent I'm fine either way @stbrody. Right now `-url` is only on `anchor-service`.

Reply:
You're right, if we change this we should change all the other flag names too.  Maybe it's not actually worth it.  New version up that puts the flag name back the way it was